### PR TITLE
feat: steps campaign application and details skeleton (Epic #1842) 

### PR DESCRIPTION
**Pull request by teodorazhelyazkova**
Skeleton of the last two steps without styles, validation, error handling etc, so we can show them to the NGO for feedback as requested in the following task https://github.com/users/gparlakov/projects/1/views/1?pane=issue&itemId=70613020

*Connected to [Epic #1842](https://github.com/podkrepi-bg/frontend/issues/1842)

## Screenshots:

- Step 2: "Campaign Application"
step2

- Step 3: "Campaign Details"
step3

### Steps to test

1. Login as normal user
2. Visit `http://localhost:3040/campaigns/application`
3. Fill in the fields and press button "Save and continue"

### CODE REVIEW

File: src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx [hunk 1]
```diff
@@ -1,3 +1,95 @@
...
+                type="text"
+                name="details.links.donationPlatform"
+                label=""
+                placeholder={t('steps.details.links.fields.donation-platforms')}
+              />
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid item xs={12}>
+          <FileUpload buttonLabel={t('steps.details.photos')} onUpload={(files) => {}} />
```
Comment (sashko9807) [inline src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx:87]: Would you place some `return` clause within the onUpload function to make linter happy

File: src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx [hunk 2]
```diff
@@ -1,3 +1,95 @@
...
+        </Grid>
+        <Grid item xs={12}>
+          <FileUpload buttonLabel={t('steps.details.documents')} onUpload={(files) => {}} />
```
Comment (sashko9807) [inline src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx:90]: Same as above.

**Review by sashko9807 — CHANGES_REQUESTED** [review]

File: src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx (hunk 2 above)
Comment (teodorazhelyazkova) [reply src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx]: Done

File: src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx (hunk 1 above)
Comment (teodorazhelyazkova) [reply src/components/client/campaign-application/steps/CampaignApplicationDetails.tsx]: Done

**Review by sashko9807 — APPROVED** [review]

**Pull request merged**